src/app/terminology/terminology-details/terminology-details.component.ts

```diff
@@ -196,39 +133,20 @@ export class TerminologyDetailsComponent implements OnInit, OnDestroy {
         );
       }
     }
-  };
+  }
 
-  formBeforeSave = () => {
+  save = () => {
```

Comment:
Might as well update this function declaration to be `save() { ... }` instead of property syntax.